### ISSUE TITLE
Finetuning the RealSR pretrained model

### ISSUE THREAD

**Issue by yxldd** [Zj-BinXia/DiffIR#20]
I want to try finetuning the pretrained model to have a better performance on my privative data, but my single gpu is not suitable for the   current finetuning program because of the distributed training mode. It's hard for me to deal with the problems on my own.  So, how can I adjust my code to run the finetuning program? I would very much like to use DiffIR to solve problems in the field of my research area.

**Comment by Zj-BinXia** [comment]
you can run "python3 DiffIR/train.py -opt options/train_DiffIRS2_GAN_x4.yml"
